netutils/config/parser.py

```diff
@@ -53,7 +53,7 @@ class BaseSpaceConfigParser(BaseConfigParser):
     # pylint: disable=abstract-method
 
     comment_chars = ["!"]
-    banner_start = ["banner", "vacant-message"]
+    banner_start = ["banner", "vacant-message", " banner", "  banner"]
```

Comment:
I don't understand this, why are we adding banner_start multiple times?

Reply:
The first one has has 1 white space and the second one has two.

Reply:
I would not think this is how we want to do this then. I would think we would understand the parent / child relationship would be how we determine it.

Reply:
Check in with @jmcgill298

Reply:
ya, that banner is a child of the group-policy, so it should be captured there. I guess for this case the banner is not multiline?